frontend/src/hooks/logo-hooks.ts

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState, useCallback } from "react";
+import ContentService from "../services/ContentService";
+
+type UseLogoHook = {
+  loadingFile: boolean;
+  logo: File | undefined;
+};
+
+export function useLogo(s3Key: string | undefined): UseLogoHook {
```

Comment:
I think is the same if you make the parameter optional `?`. Removes the need to specify `undefined` as an accepted data type. just fyi, no need to change. 
```js
function useLogo(s3Key?: string) {
  
}
```

Reply:
ah nice. That is cleaner. But mine is more verbose. I guess we'll stick with mine?